fhir-persistence-cassandra/pom.xml

```diff
@@ -109,4 +109,41 @@
         </dependency>
     </dependencies>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>shade</goal>
+                        </goals>
+                        <configuration>
+                            <shadedArtifactAttached>true</shadedArtifactAttached>
+                            <shadedClassifierName>cli</shadedClassifierName>
```

Comment:
I assume this is used to lay down some kind of schema in cassandra?
I wonder if that should be broken out like we did for fhir-persistence-schema...

Reply:
be sure to exclude this CLI from the appscan, it'll blow out the package size.

Reply:
We could definitely pull this out into a separate project to keep the jar clean. The app will have two roles: 1. boostrapping the datastore (create schema); and 2. Implementing the reconciliation process.

Reply:
Excluded the jar in build/security/asoc.sh

Reply:
Also I've created a new project to host just the CLI part, as we have for fhir-persistence-schema